_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -889,16 +889,206 @@ Terraform::
   Comprehensive Guide to Terraform], https://training.gruntwork.io/p/terraform[A Crash Course on Terraform], and
   link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library[How to use the Gruntwork Infrastructure as Code Library].
 
+Terragrunt::
+  This guide uses https://terragrunt.gruntwork.io/[Terragrunt] to configure the infrastructure as code. To get familiar
+  with Terragrunt, explore the https://terragrunt.gruntwork.io/docs/#features[features], read the https://terragrunt.gruntwork.io/docs/getting-started/quick-start/[guides],
+  or dive into the https://terragrunt.gruntwork.io/docs/[documentation].
+
+Code repository::
+  You will need to initialize an `infrastructure-live` repository to contain all of the Terragrunt configuration code for your
+  infrastructure. You may use the https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/v0.41.4/examples/for-production/infrastructure-live[`for-production` example code] to start with.
+
 Keybase (optional)::
   As part of this guide, you will create IAM users, including, optionally, credentials for those IAM users. If you
   choose to create credentials, those credentials will be encrypted with a PGP key. You could provide the PGP keys
   manually, but a more manageable option may be to have your team members to sign up for https://keybase.io[Keybase],
   create PGP keys for themselves, and then you can provide their Keybase usernames, and the PGP keys will be retrieved
   automatically.
 
+
+=== Prepare your `infrastructure-live` repository
```

Comment:
Using ``` makes the font white, making it hard to read
<img width="808" alt="Screen Shot 2021-06-15 at 12 07 17" src="https://user-images.githubusercontent.com/9384479/122034882-3d49b900-cdd2-11eb-825c-e8bc987ef3b6.png">

Reply:
Ah, yes. I meant to fix this. I think it's a bug in the CSS, where we always set `code` to have no background but make the foreground color almost white.

Reply:
Pushed a fix: 698610fe..77f011ce